renovate.json

```diff
@@ -10,19 +10,21 @@
       "matchDepTypes": ["devDependencies"],
       "matchUpdateTypes": ["patch", "minor"],
       "schedule": ["before 4am on Monday"],
-      "groupName": "non-major-dev-dependencies"
+      "groupName": "non-major-dev-dependencies",
+      "automerge": true
     },
     {
       "matchDepTypes": ["dependencies"],
       "matchUpdateTypes": ["patch", "minor"],
       "schedule": ["before 4am on Monday"],
-      "groupName": "non-major-dependencies"
+      "groupName": "non-major-dependencies",
+      "automerge": true
     }
   ],
   "rangeStrategy": "bump",
   "lockFileMaintenance": {
-    "enabled": false
+    "enabled": true
```

Comment:
Any reason to turn this on?

Reply:
(Reverted, see here: https://github.com/api3dao/oev-searcher/pull/327#discussion_r1709732933 )